python/kserve/pyproject.toml

```diff
@@ -43,8 +43,8 @@ python-dateutil = "^2.8.0"
 numpy = "^1.23.5"
 psutil = "^5.9.0"
 ray = { version = "~2.10.0", extras = ["serve"] }
-grpcio = "^1.49.1"
-protobuf = "^3.19.0"
+grpcio = "^1.64.1"
+protobuf = "^5.27.1"
```

Comment:
Is this backwards compatible and works with the older triton inference runtime version ?